### PR TITLE
OSDOCS-10773: Combine “understanding ROSA” page with “What is ROSA” for HCP

### PR DESCRIPTION
**Pull request by jneczypor**
[OSDOCS-10773](https://issues.redhat.com//browse/OSDOCS-10773): Combine “understanding ROSA” page with “What is ROSA” for HCP

Version(s):
4.16+

Issue:
https://issues.redhat.com/browse/OSDOCS-10773

Link to docs preview:
Preview unavailable. This is for the ask-of-yet unpublished HCP docs set.

QE review:
- [ ] QE has approved this change.

Additional information:

### CODE REVIEW

File: rosa_architecture/about-hcp.adoc [hunk 1]
```diff
@@ -1,23 +1,43 @@
...
+
+== Key features of {hcp-title}
 
-* Customers can upgrade the control plane and machine pools separately, which means they do not have to shut down the entire cluster during upgrades.
+* *Cluster node scaling:* {hcp-title} requires a minimum of only two nodes, making it ideal for smaller projects while still being able to scale to support larger projects and enterprises. Easily add or remove compute nodes to match resource demand. Autoscaling allows you to automatically adjust the size of the cluster based on the current workload. See xref:../rosa_cluster_admin/rosa_nodes/rosa-nodes-about-autoscaling-nodes.adoc#rosa-nodes-about-autoscaling-nodes[About autoscaling nodes on a cluster] for more details.
```
Comment (EricPonvelle) [inline rosa_architecture/about-hcp.adoc:27]: Without seeing the preview, we need to ensure that we come back and test the links. Xrefs only work within the same topic map. If these xref'd topics are within that map, we have to use links.

File: rosa_architecture/about-hcp.adoc [hunk 2]
```diff
@@ -1,23 +1,43 @@
...
+* *Rapid provisioning time:* Provisioning time is approximately 10 minutes.
+* *Continued cluster operation during upgrades:* Customers can upgrade the control plane and machine pools separately, which means they do not have to shut down the entire cluster during upgrades.
+* *Native AWS service:* Access and use Red{nbsp}Hat OpenShift on-demand with a self-service onboarding experience through the AWS management console.
+* *Flexible, consumption-based pricing:* Scale to your business needs and pay as you go with flexible pricing and an on-demand hourly or annual billing model.
+* *Single bill for Red{nbsp}Hat OpenShift and AWS usage:* Customers will receive a single bill from AWS for both Red{nbsp}Hat OpenShift and AWS consumption.
+* *Fully integrated support experience:* Installation, management, maintenance, and upgrades are performed by Red{nbsp}Hat site reliability engineers (SREs) with joint Red{nbsp}Hat and Amazon support and a 99.95% service-level agreement (SLA). See the xref:../support/getting-support.adoc#getting-support[ROSA support documentation] for more details.
```
Comment (EricPonvelle) [inline rosa_architecture/about-hcp.adoc:34]: Xref

File: rosa_architecture/about-hcp.adoc [hunk 3]
```diff
@@ -1,23 +1,43 @@
...
 
-* The underlying control plane infrastructure is fully managed. Control plane components, such as the API server and etcd database, are hosted in a Red{nbsp}Hat-owned AWS account.
+image::291_OpenShift_on_AWS_Intro_1122_docs.png[{product-title}]
 
-* Provisioning time is approximately 10 minutes.
+ROSA uses AWS Security Token Service (STS) to obtain credentials to manage infrastructure in your AWS account. AWS STS is a global web service that creates temporary credentials for IAM users or federated users. ROSA uses this to assign short-term, limited-privilege, security credentials. These credentials are associated with IAM roles that are specific to each component that makes AWS API calls. This method aligns with the principals of least privilege and secure practices in cloud service resource management. The ROSA command line interface (CLI) tool manages the STS credentials that are assigned for unique tasks and takes action on AWS resources as part of OpenShift functionality. For a more detailed explanation, see xref:../rosa_architecture/cloud-experts-rosa-hcp-sts-explained.adoc#cloud-experts-rosa-hcp-sts-explained[AWS STS and ROSA with HCP explained].
```
Comment (EricPonvelle) [inline rosa_architecture/about-hcp.adoc:23]: Xref

File: rosa_architecture/about-hcp.adoc [hunk 4]
```diff
@@ -1,23 +1,43 @@
...
+* *AWS service integration:* AWS has a robust portfolio of cloud services, such as compute, storage, networking, database, analytics, and machine learning. All of these services are directly accessible through ROSA. This makes it easier to build, operate, and scale globally and on-demand through a familiar management interface.
+* *Maximum availability:* Deploy clusters across multiple availability zones in supported regions to maximize availability and maintain high availability for your most demanding mission-critical applications and data.
+* *Optimized clusters:* Choose from memory-optimized, compute-optimized, or general purpose EC2 instance types with clusters sized to meet your needs.
+* *Global availability:* Refer to the xref:../rosa_architecture/rosa_policy_service_definition/rosa-service-definition.adoc#rosa-sdpolicy-regions-az_rosa-service-definition[product regional availability page] to see where ROSA is available globally.
```
Comment (EricPonvelle) [inline rosa_architecture/about-hcp.adoc:38]: Xref

**Review by EricPonvelle — COMMENTED** [review]
Reads well enough for me. I just want to make sure we check the xrefs.

**Comment by jneczypor** [comment]
Hi @yuwang-RH - Can I please get QE review of this PR? Thank you!

**Review by EricPonvelle — COMMENTED** [review]
/lgtm

**Pull request merged**